packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -57,14 +57,22 @@ export const messages = {
   parameter_gatherer_enter_lwc_name:
     'Enter desired Lightning Web Component (Press Enter to confirm or Esc to cancel)',
   parameter_gatherer_enter_username_name: 'Enter target username',
+  parameter_gatherer_enter_source_org:
+    'Enter source org to clone or use default value from sandbox-def.json file',
```

Comment:
Is this always a sandbox? and assuming they entering the sandboxName here? If my assumptions are correct:

Enter the source sandbox name to clone, or use the default value in  the sandbox-def.json file